### DISCUSSION TITLE
What is causing the error Cannot find name 'useSupabaseClient'. when using Nuxt 3 and Supabase?

### DISCUSSION THREAD

**Discussion by adolfusadams** [supabase/supabase#20486] [Questions]
I have been trying to resolve this issue without any success, the details of which I have posted here https://stackoverflow.com/questions/77808228/what-is-causing-the-error-cannot-find-name-usesupabaseclient-when-using-nuxt.

Can anyone tell me what is causing this issue and how can it be resolved?

**Comment by GaryAustin1** [comment]
I commented in your post in Discord.  That is a Nuxt function not a Supabase function.  You should also ask in their forums.
